src/Microsoft.Extensions.Http.Polly/HttpRequestMessageExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Http;
+using System;
+using System.Net.Http;
+
+namespace Polly
+{
+    /// <summary>
+    /// Extension methods for <see cref="HttpRequestMessage"/> Polly integration.
+    /// </summary>
+    public static class HttpRequestMessageExtensions
+    {
```

Comment:
Awesome to see the `Context` support going in; great that we can add xtn methods to `HttpRequestMessage` to hide the details.